src/utils/dates.js

```diff
@@ -145,12 +153,18 @@ let dates = {
     return dates.startOf(new Date(), 'day')
   },
 
-  yesterday() {
-    return dates.add(dates.startOf(new Date(), 'day'), -1, 'day')
+  tomorrow(date) {
+    const tomorrow = new Date(date.valueOf())
```

Comment:
can we use `getTime()` or `+date` instead of `valueOf`

Reply:
using `getTime` everywhere now